### DISCUSSION TITLE
Why does Image component not add width and height to the <img/>??

### DISCUSSION THREAD

**Discussion by amir-mln** [vercel/next.js#22904] [Help]
Hello

I'm using the image component instead of html img tag for the first time; after auditing the site with lighthouse, I realized that the component does not actually add the width and height to the element's attributes and lighthouse diagnoses it as a problem.

so as I've already stated in the title. Why does Image component not add width and height to the `<img>`??

![2021-03-09_17-50-22](https://user-images.githubusercontent.com/69141584/110485204-a4374200-8100-11eb-9b29-48a612f34c27.jpg)

**Comment by qpuolet** [comment]
I have same issue! Need to be improved

**Comment by Sajletry** [comment]
It seems like it not affect to the CLS, but still it will be better for SEO.
